addons/QuickSceneRunner/SceneRunner.gd

```diff
@@ -53,7 +53,8 @@ func update_selected(scene: Control):
 	ProjectSettings.set_setting(SELECTED_SCENE_SETTING, scene.get_index())
 	ProjectSettings.save()
 
-func run_scene(scene: Control):
+func run_scene(scene: Control=null):
+	if scene == null: scene = shortcut_group.get_pressed_button().get_parent().get_parent().get_parent()
```

Comment:
This will fail if there is no pressed button.
Also for readability:
```suggestion
	if scene == null:
		scene = shortcut_group.get_pressed_button().get_parent().get_parent().get_parent()

```